### PR TITLE
[IA-852] Copy FAQ BANCOMAT Pay

### PR DESCRIPTION
**Pull request by mariateresaventura**
Adding copy FAQ BANCOMAT Pay

### CODE REVIEW

**Comment by thisisjp** [comment]
@fabriziofff can you help us with the JSON of this PR? 🙏 💙

File: contextualhelp/data.json [hunk 1]
```diff
@@ -1048,6 +1137,15 @@
           }
         ]
       },
+      {
+        "route_name": "WALLET_ONBOARDING_BPAY_SEARCH_AVAIABLE_USER_ACCOUNT",
```
Comment (thisisjp) [inline contextualhelp/data.json:1141]:
```suggestion
        "route_name": "WALLET_ONBOARDING_BPAY_SEARCH_AVAILABLE_USER_ACCOUNT",
```

File: contextualhelp/data.json [hunk 2]
```diff
@@ -1048,6 +1137,15 @@
           }
         ]
       },
+      {
+        "route_name": "WALLET_ONBOARDING_BPAY_SEARCH_AVAILABLE_USER_ACCOUNT",
+        "faqs": [
+          {
+            "title": "I cannot add BANCOMAT Pay as a payment method.",
+            "body": "If you have problems adding your ATM Pay account to IO, we suggest you:\n1. Check that your device is connected to a network.\n2. Make sure the service is active. You can do this via your home banking or your bank app, or via the ATM Pay app.\n\nIf you have done all the steps correctly and the problem is not yet solved contact your bank."
```
Comment (thisisjp) [inline contextualhelp/data.json:1145]:
```suggestion
            "body": "If you have problems adding your BANCOMAT Pay account to IO, we suggest you:\n1. Check that your device is connected to a network.\n2. Make sure the service is active. You can do this via your home banking or your bank app, or via the BANCOMAT Pay app.\n\nIf you have done all the steps correctly and the problem is not yet solved contact your bank."
```

**Review by thisisjp — APPROVED** [review]

**Comment by thisisjp** [comment]
@fabriziofff I think I solved all the issues with the JSON file, I got a green check 🥳 Could you please approve the PR so we can publish the FAQs? 🙏

**Review by fabriziofff — APPROVED** [review]

**Pull request merged**